### ISSUE TITLE
Updated cpp examples for new c++/WinRT

### ISSUE THREAD

**Issue by tdwoods** [Microsoft/Windows-universal-samples#927]
Is there going to be an update to the samples to incorporate the changes made to c++/WinRT development? I am trying to learn how to create a DeviceWatcher in c++/WinRT but the samples are all in c++/CX.

**Comment by techjogger** [comment]
I second this request for the current C++/CX samples to be ported to C++/WinRT! Specifically, I would like to see the Simple3DGameXaml ported to WinRT and a NEW C++/WinRT App using  C#/XAML  from the Community toolkit to display UI.

**Comment by oldnewthing** [comment]
Yes, work items have been filed against all C++/CX samples to port to C++/WinRT.
